### ISSUE TITLE
Lighthouse report for AMP shows low score

### ISSUE THREAD

**Issue by anuragphadke** [GoogleChrome/lighthouse#8932]
We have a fully valid AMP page, however, while running a lighthouse report, the score ranks in 60's. I tried testing with AMP CDN cache, same issue.

Is there any way to get actual score when the page goes live? https://github.com/GoogleChrome/lighthouse/issues/4490#issuecomment-364437127 says that this is an inherent limitation on how score is calculated for AMP pages, wanted to see if there's any update given all the changes in LHR.
#### Provide the steps to reproduce
1. Run LH on https://mutesix-pfpmarketing-com.cdn.ampproject.org/c/s/mutesix.pfpmarketing.com/amp/spend-10k-day-podcast-podcast-episode-2-find-right-attribution-model-business.html
OR
https://mutesix.pfpmarketing.com/amp/spend-10k-day-podcast-podcast-episode-2-find-right-attribution-model-business.html

#### What is the current behavior?
Score ~60

#### What is the expected behavior?
Score > 90

#### Environment Information
**Related issues**
https://github.com/GoogleChrome/lighthouse/issues/4490

**Comment by patrickhulce** [comment]
Thanks @anuragphadke!

This is what I'm seeing in latest lighthouse. 

![image](https://user-images.githubusercontent.com/2301202/57574141-223d3b80-73f9-11e9-91ef-9b7644a43d76.png)

Would you be able to share the artifacts of the run where you're seeing a score of 60? (run `lighthouse --save-assets <url>` and upload the `trace.json` and `devtoolslog.json` files that get create)

**Comment by anuragphadke** [comment]
@patrickhulce I am running the report via https://web.dev, report URL: https://lighthouse-dot-webdotdevsite.appspot.com/lh/html?url=https://mutesix-pfpmarketing-com.cdn.ampproject.org/c/s/mutesix.pfpmarketing.com/amp/spend-10k-day-podcast-podcast-episode-2-find-right-attribution-model-business.html

requested dumps `lighthouse --save-assets https://mutesix-pfpmarketing-com.cdn.ampproject.org/c/s/mutesix.pfpmarketing.com/amp/spend-10k-day-podcast-podcast-episode-2-find-right-attribution-model-business.html`


trace.json: https://gist.github.com/anuragphadke/f0ca418b0aff97b537815a9cc8a1997d
devtoolslog.json: https://gist.github.com/anuragphadke/5f5690f69a2b3f6568ed2b53a52095ec

Looks like there is inconsistency when running locally vs web.dev?
The FCP and FMP numbers even after running locally are still high at 3-secs and the recommendation in `Preload key requests` should probably be different for valid AMP pages.

I got a 90 score when running it locally.
![image](https://user-images.githubusercontent.com/349486/57574271-b5bb4000-73ea-11e9-81fa-706f6d38ac6f.png)

**Comment by patrickhulce** [comment]
Ah thanks for the additional info @anuragphadke!

cc @exterkamp FCP looks similar but JS execution time is much higher in LR perhaps this cpuSlowdownMultiplier rearing its head?

**Comment by anuragphadke** [comment]
@patrickhulce good point on FCP, one question: Shouldn't FCP for AMP pages be lot smaller though? When I click on AMP link from Google search result, the page definitely seems to be loading in well under a second.

**Comment by patrickhulce** [comment]
@anuragphadke when the user is coming from the search results page FCP will absolutely be lower with an AMP page, but Lighthouse is measuring the page as visited by a user navigating directly to the page.

Other circumstances like the previous page precaching/prerendering aren't reflected anywhere in the core report. (great potential plugin idea though? 😃)

**Comment by exterkamp** [comment]
@patrickhulce I'll take a look at that JS exec time, see if I can't attribute it to the slowdown multiplier, or something within LR.

**Comment by erakor** [comment]
Re-upping this thread as we're experiencing the same issues.

Fully AMP compliant pages give a 48 Performance score

![image](https://user-images.githubusercontent.com/352881/88322044-1aac3e80-ccee-11ea-8002-57f7b7f8e1c6.png)

The diagnostic critical points seem to be all related to AMP and the JS.

![image](https://user-images.githubusercontent.com/352881/88322157-4cbda080-ccee-11ea-800e-dcf08d843121.png)

Is there anything anyone sees here we can on our end do to fix those issues? 
The report can be seen [here](https://lighthouse-dot-webdotdevsite.appspot.com//lh/html?url=https%3A%2F%2Fwww.understood.org%2Fen%2Ffamily%2Fmanaging-everyday-challenges%2Fdaily-expectations-child%2Fscreen-time-over-can-child-stop)

**Comment by geogramdotcom** [comment]
Same issues on our sites. We were getting >95 for performance and now getting ~65.  According to the performance opportunities, it is indicating _"Remove unused JavaScript"_ from numerous javascript files served from amp cdn. 


<img width="786" alt="Screen Shot 2020-08-06 at 12 58 00 PM" src="https://user-images.githubusercontent.com/35526690/89576765-90330700-d7e4-11ea-8fee-e539e052166f.png">

**Comment by patrickhulce** [comment]
@geogramdotcom just to be clear that audit does not affect the score. It helps suggest what *is* dragging down the score (namely that the page is using too many AMP modules), but the audit itself appearing does not affect the score in any way.

**Comment by homanp** [comment]
Experiencing drastically dropping scores the last couple of weeks for our valid AMP Pages as well even though nothing has changed on our side. Definitely something strange going on with both web.dev and pagespeed insights.
